test_files/extend_and_implement/extend_and_implement.js

```diff
@@ -0,0 +1,30 @@
+/**
+ *
```

Comment:
I wonder why we get this extra whitespace here.

Reply:
Huh. Maybe we parse @extends incorrectly? I now also notice we emit `\\@extends`, not `\@extends`. I'll take a look later.